### PR TITLE
[Matrix] some cleanups

### PR DESCRIPTION
**Pull request by AlwinEsch**
That I bring this request has primarily to do with Azure Test, somehow it no longer runs at https://github.com/xbmc.

This cleanup some part on the addons:

Commit 1:
-----
**add asserts for icon and remove nofanart on addon.xml**
This related to add asserts for icon and remove nofanart on addon.xml

Commit 2:
-----
**update addon.xml text language set**

As standard this now `lang="en_GB"` for Kodi

Commit 3:
----------
**update language folder from English to resource.language.en_gb**

Also here kodi have now the `resource.language.en_gb` as standard and the old has only worked with fallback.

Commit 4:
-----------
**update transifex set on strings.po**

To have them on right links and with correct language for Transifex

Commit 5:
-----------
**update .gitignore**

To have equal with other addons and to prevent much not wanted parts.

Commit 6:
-----------
**update debian build**

Then equal to others and better to test with https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-addon-dev-kit/tools/debian-addon-package-test.sh

### CODE REVIEW

**Comment by AlwinEsch** [comment]
About Azure was a bug brought in, found this related to them:
- https://github.com/hdmf-dev/hdmf/pull/313
- https://developercommunity.visualstudio.com/content/problem/947762/github-pull-request-not-triggering-azure-pipelines.html

**Review by garbear — APPROVED** [review]

**Comment by AlwinEsch** [comment]
Added two other commits, the first about https://github.com/kodi-pvr/pvr.hts/issues/441 and to have visible on html git tabs.
The second add also the URL and license name to addon.xml, to have on webview.

About Azure problem, them fixed and hack commit removed.

File: game.libretro/addon.xml.in [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
        library_@PLATFORM@="@LIBRARY_FILENAME@">
   </extension>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Libretro compatibility layer for Kodi's Game API</summary>
-    <description>This add-on provides a wrapper for libretro cores, allowing them to be loaded as game add-ons.</description>
+    <summary lang="en_GB">Libretro compatibility layer for Kodi's Game API</summary>
+    <description lang="en_GB">This add-on provides a wrapper for libretro cores, allowing them to be loaded as game add-ons.</description>
     <platform>@PLATFORM@</platform>
-    <nofanart>true</nofanart>
+    <license>GPL-2.0+</license>
```
Comment (garbear) [inline game.libretro/addon.xml.in:15]: If we're using SPDX like in cpp sources this would be `GPL-2.0-or-later`

Comment (AlwinEsch) [reply game.libretro/addon.xml.in:15]: Ah thanks and sorry, was coming by them https://github.com/kodi-pvr/pvr.hts/pull/447#discussion_r398092817, I change soon.

**Pull request merged**